src/main/java/it/gov/pagopa/pu/bff/connector/debt_position/DebtPositionServiceImpl.java

```diff
@@ -34,4 +38,10 @@ public PagedModelDebtPositionView getDebtPositionViews(
   public DebtPositionDTO getDebtPosition(Long debtPositionId, String accessToken) {
     return client.getDebtPosition(debtPositionId, accessToken);
   }
+
+  @Override
+  public CollectionModelDebtPosition getDebtPositionByDebtPositionTypeOrgId(Long debtPositionTypeOrgId,
+    String accessToken) {
+    return debtPositionSearchClient.getDebtPositionByDebtPositionTypeOrgId(debtPositionTypeOrgId,accessToken);
```

Comment:
let's not fetch all DebtPosition based on debtPositionTypeOrgId!
you are interested to know if there is at least one, this could return millions of data!
Thus, or you expose this query paged, asking for first page of size 1, or just return a count